pkg/bpf/bpf_linux.go

```diff
@@ -301,7 +305,11 @@ func ObjPin(fd int, pathname string) error {
 
 // ObjGet reads the pathname and returns the map's fd read.
 func ObjGet(pathname string) (int, error) {
-	pathStr := syscall.StringBytePtr(pathname)
+	pathStr, err := syscall.BytePtrFromString(pathname)
+	if err != nil {
+		return -1, fmt.Errorf("Unable to convert pathname %q to byte pointer: %w", pathname, err)
```

Comment:
I think that `-1` should be `0` here considering it's the default value and how it's used on non-nil return path in this function.

Reply:
Thanks, done.